Mathlib/Probability/ProbabilityMassFunction/Binomial.lean

```diff
@@ -0,0 +1,47 @@
+/-
+Copyright (c) 2023 Joachim Breitner. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joachim Breitner
+-/
+import Mathlib.Probability.ProbabilityMassFunction.Constructions
+import Mathlib.Data.Nat.Choose.Multinomial
+import Mathlib.Tactic.FinCases
+
+/-!
+# The binomial distribution
+
+This file defines the probability mass function of the binomial distribution, and proves
+it to be equal to `Pmf.bernoulli` for `n = 1`.
+-/
+
+noncomputable section
```

Comment:
I'd rather have a single `noncomputable` in front of the definition of binomial. Also here you open a section that you never close. Let's remove that line.
```suggestion
```